app/src/shared/utils/useMountedCallback.js

```diff
@@ -0,0 +1,13 @@
+import { useCallback } from 'react'
```

Comment:
maybe should move all shared hooks into `shared/hooks` instead of `shared/utils`.

Also thinking this could be more clearly named, something like, `useOnlyIfMountedCallback()` or something, currently sounds like it's a callback that fires when the component is mounted.

Reply:
Moved the hooks and renamed that one in https://github.com/streamr-dev/streamr-platform/pull/590/commits/9672fcf28715a13bfab5877bd097f434873e796f.